debian/control

```diff
@@ -69,6 +69,9 @@ Build-Depends: automake,
                librabbitmq-dev,
                librdkafka-dev,
                luarocks,
+               libthrift-0.13.0 <pkg.ceph.jaeger>,
```

Comment:
> cmake: delete Buildyaml, since distro suppilies v0.6 this is not needed

the title should read

>
> cmake, debian:  use prebuilt dependencies from distro
>
> \<put more details here\>
>

Reply:
i think `libthrift-dev (>= 0.13.0) <pkg.ceph.jaeger>,` would suffice.